icaruscode/Timing/PMTTimingCorrectionsProvider.h

```diff
@@ -19,32 +19,24 @@
 #include "fhiclcpp/types/Sequence.h"
 #include "cetlib_except/exception.h"
 
-#include "lardata/DetectorInfoServices/DetectorClocksService.h"
-
+// Local
 #include "icaruscode/Timing/PMTTimingCorrections.h"
 
 // Database interface helpers
-#include "wda.h"
+#include "larevt/CalibrationDBI/Providers/DBFolder.h"
```

Comment:
I don't see anything in this header that needs `DBFolder`. It is (correctly) included into the source file already.
```suggestion
```